cmd/helper.go

```diff
@@ -29,3 +29,14 @@ func getAWSConfiguration(logger *slog.Logger, roleArn string, sessionName string
 
 	return cfg, nil
 }
+
```

Comment:
I would move these functions to another package than `cmd`. And use a more appropriate filename.